lib/queue.js

```diff
@@ -783,6 +783,7 @@ Queue.prototype.processJob = function(job){
 
     return job.moveToCompleted(result).then(function(){
       _this.emit('completed', job, result, 'active');
+      return null;
```

Comment:
Maybe. I just applied the same fix I saw here: https://github.com/OptimalBits/bull/commit/1274f3ced5024c493c84bc4bb5f2246a28b44209
Also, there were others`return null;` in this file before this modification. We could replace all of them if you want.

We would be returning a promise. Can this be problematic to support both callbacks and promise based behavior from outside the package?